sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.aest.controller.getStudentFileUploads.e2e-spec.ts

```diff
@@ -0,0 +1,71 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
```

Comment:
As per the file naming convention that we follow, controller method name must match the E2E file name.
![image](https://github.com/user-attachments/assets/5f1b46d3-f973-4de7-8563-d3834d948dfb)